nsxt/resource_nsxt_policy_group.go

```diff
@@ -72,6 +72,13 @@ func resourceNsxtPolicyGroup() *schema.Resource {
 				Elem:        getConjunctionSchema(),
 				Optional:    true,
 			},
+			"identity_group": {
```

Comment:
Should this be part of criteria? Can it be part of AND/OR clauses with other expressions?

Reply:
Nope this is different from Criteria. Criteria is added in expression field in the actual api call. But identity_group is only supported in extended_expression field. In fact, identity_group is the only resource supported in extended_expression now, and is limit to one at most. From api doc:

"Extended Expression allows additional higher level context to be
specified for grouping criteria. (e.g. user AD group)
...
Current version only supports a single IdentityGroupExpression. In the
future, this might expand to support other conjunction and non-conjunction
expression."

On second thought I think it would be best if we introduce an extended_criteria field, and the schema of it should be similar to criteria, only that currently we support one single identity_group in it.

Reply:
> Please also add the new structure to the group doc

Sure. Will update the doc when code is finalized. Thanks.

Reply:
Thanks for the explanation :) Is there an AND relationship with "old" expression?

Reply:
> Thanks for the explanation :) Is there an AND relationship with "old" expression?

Yes Anna, according to api doc, "Extended expression are implicitly AND with expression". I will add this in Group doc.